content/docs/reference/routes/from.mdx

```diff
@@ -16,21 +17,62 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-**From** is the externally accessible URL for the proxied request.
+The **From** route is the externally accessible URL for a proxied HTTP request.
 
 Specifying `tcp+https` for the scheme enables [TCP proxying](/docs/capabilities/tcp/) support for the route. You may map more than one port through the same hostname by specifying a different `:port` in the URL.
 
 ## How to configure
 
-The `URL` must contain a **scheme** and **hostname** and cannot contain a path.
+The From URL must contain a **scheme** and **hostname**. It can't contain a path.
+
+### Port mapping behavior
+
+In v0.26, we've updated the default route matching behavior so that From routes defined _without_ a specific port in the route URL will match a request on any port.
```

Comment:
Likewise here, let's be precise with our terminology: "from route" doesn't make sense to me, and neither does "route URL" (there are two URLs, the From URL and the To URL).

I also think it might make sense to describe the behavior fully up front, and then proceed with the examples.

```suggestion
In v0.26, we've updated the route matching behavior. If you do not specify a port number in a route's From URL, the route will match incoming requests with any port number (or no port number). If you do specify a port number, that route will match only requests containing that specific port number.
```